base/abstractarray.jl

```diff
@@ -118,18 +118,6 @@ ndims(::AbstractArray{T,N}) where {T,N} = N
 ndims(::Type{AbstractArray{T,N}}) where {T,N} = N
 ndims(::Type{T}) where {T<:AbstractArray} = ndims(supertype(T))
 
-"""
-    length(A::AbstractArray) -> Integer
-
-Returns the number of elements in `A`.
-
-```jldoctest
-julia> A = ones(3,4,5);
```

Comment:
Right, the `;` confused me (I thought that was a matrix). I've added a new one and adapted the existing one so that should be even more explicit.